Memeolist/Memeolist/view/list-meme/MemesTableViewCell.swift

```diff
@@ -8,12 +8,15 @@ class MemesTableViewCell: UITableViewCell {
 
     @IBOutlet var likesLabel: UILabel!
     @IBOutlet var memeImageView: UIImageView!
+    @IBOutlet var user:UILabel!
+    @IBOutlet var avatar:UIImageView!
 
     func configure(with meme: AllMemesQuery.Data.AllMeme) {
         memeId = meme.id
         let url = URL(string: meme.photourl)
         memeImageView.kf.setImage(with: url, placeholder: UIImage(named: "loading"))
-        likesLabel?.text = "0 likes" //"\(meme.votes ?? 0) likes"
+        likesLabel?.text = "\(meme.likes) likes"
+        user?.text = meme.owner ?? "David Duchovny"
```

Comment:
owner is required and it should be always set. It is not?

Reply:
You're right, it is.

Reply:
Let's remove  ![](https://i.pinimg.com/236x/dc/75/4a/dc754a889eada571e7178210176db016--david-duchovny.jpg)